### ISSUE TITLE
Chose the unit

### ISSUE THREAD

**Issue by matthw** [kouinkouin/when-he-leaves#12]
F5 is good for random unit change, but it would be good to be able to chose a specific display unit (from a drop down or whatever fancy feature modern php can offer)

**Comment by Cryde** [comment]
Good idea for the example page but don't add this in the plugin itself. Thanks.
